On buildozer the error says:
failed to encrypt sops data key with pgp: gpg: WARNING: unsafe permissions on homedir '/home/buildozer/.gnupg'\ngpg: 'invalid' is not a valid long keyID

--- a/pgp/keysource_test.go
+++ b/pgp/keysource_test.go
@@ -300,7 +300,8 @@ func TestMasterKey_encryptWithGnuPG(t *testing.T) {
 		key := NewMasterKeyFromFingerprint("invalid")
 		err := key.encryptWithGnuPG([]byte("invalid"))
 		assert.Error(t, err)
-		assert.ErrorContains(t, err, "failed to encrypt sops data key with pgp: gpg: 'invalid' is not a valid long keyID")
+		assert.ErrorContains(t, err, "failed to encrypt sops data key with pgp:")
+		assert.ErrorContains(t, err, "gpg: 'invalid' is not a valid long keyID")
 	})
 
 }
